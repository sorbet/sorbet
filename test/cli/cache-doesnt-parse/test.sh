#!/bin/bash
dir=$(mktemp -d)
cleanup() {
    rm -r "$dir"
}
trap cleanup EXIT
set -e

mkdir "$dir/cache"
cat >> "$dir/test.rb" <<EOF
puts("hi".)
EOF
main/sorbet --silence-dev-message --cache-dir "$dir/cache" "$dir/test.rb" -q 2>&1
echo "should not cache erroring trees"
main/sorbet --silence-dev-message --cache-dir "$dir/cache" "$dir/test.rb" 2>&1 | grep -v "test.rb" # strip file name as it has tmp stuff in it
