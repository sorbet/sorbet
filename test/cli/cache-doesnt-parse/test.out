No errors! Great job.
should not cache erroring trees
     1 |puts("hi".)
                  ^
Errors: 1
